capellambse/model/common/element.py

```diff
@@ -617,13 +623,15 @@ def __html__(self) -> markupsafe.Markup:
         escape = markupsafe.Markup.escape
         fragments = ['<ol start="0" style="text-align: left;">']
         for i in self:
-            fragments.append(
-                "<li>"
-                f"<strong>{escape(type(i).__name__)}</strong>"
-                f" &quot;{escape(i.name)}&quot;"
-                f" ({escape(i.uuid)})"
-                "</li>"
-            )
+            if hasattr(i, "_short_html_"):
+                content = i._short_html_()
+            else:
+                content = (
+                    f"<strong>{escape(type(i).__name__)}</strong>"
+                    f" &quot;{escape(i.name)}&quot;"
+                    f" ({escape(i.uuid)})"
+                )
+            fragments.append(f"<li>{content}</li>")
```

Comment:
It should be safe to drop the condition and `else` branch here entirely and just assume that everything has a `_short_html_` method, because `_short_html_` is introduced into the hierarchy by `GenericElement` and everything yielded by `iter(self)` will be a subclass of that.

```suggestion
            fragments.append(f"<li>{i._short_html_()}</li>")
```